uvloop/loop.pyx

```diff
@@ -821,13 +824,17 @@ cdef class Loop:
                 except Exception as ex:
                     if not fut.cancelled():
                         fut.set_exception(ex)
+                    __send_trace(TRACE_DNS_REQUEST_END, None)
```

Comment:
Some questions regarding this

- do we have to use the same trace for the happy path and for the none happy path?
- do we have to pass as a parm the exception?

I'm more inclined to think that decoupling both signals to two different things will help us to make the interface cleaner, perhaps having `__send_trace(TRACE_DNS_REQUEST_END_EX, ex)`.